openerp/api.py

```diff
@@ -74,6 +77,19 @@
 INHERITED_ATTRS = ('_returns',)
 
 
+class Params(object):
+    def __init__(self, args, kwargs):
+        self.args = args
+        self.kwargs = kwargs
+    def __str__(self):
+        params = []
+        for arg in self.args:
+            params.append("%r" % arg)
```

Comment:
if `arg` happen to be a `tuple` with len != 1, this will fail. Either you use 
```python
repr(arg)
```
or
```python
"%r" % (arg,)
```